lib/jwtLib/generateToken.ts

```
@@ -0,0 +1,12 @@
+import jwt from "jsonwebtoken";
+type UserAuthDetails = {
+  email: string;
+  id: string;
+};
+export default function generateJwtToken({ id, email }: UserAuthDetails) {
+  const JWT_SECRET = process.env.JWT_SECRET;
+  if (!JWT_SECRET) return null;
```

Maybe you could throw an error here

---

Made the change here. throwing an error, which will get caught by the try catch around the calling method